frontend/src/App.jsx

```diff
@@ -1,13 +1,24 @@
 import React, { useState, useEffect } from "react";
 import axios from "axios";
-import Gametab from "./components/game/Gametab";
-import "./App.css";
-import ReminderTab from "./components/reminder/ReminderTab";
-import JournalTab from "./components/journal/JournalTab";
+import Gametab from "./components/game/Gametab"
+import './App.css';
+import ReminderTab from './components/reminder/ReminderTab';
+import JournalTab from './components/journal/JournalTab';
+import { nanoid } from 'nanoid';
 import AuthModule from "./components/auth/AuthModule";
 
 function App() {
-  const [viewTab, setViewTab] = useState("");
+  const [viewTab, setViewTab] = useState("journal");
+  const [reminderList, setReminderList] = useState([]);
+  const [modal, setModal] = useState(false);
+  const [activeItem, setActiveItem] = useState({
```

Comment:
this looks like its from before stuff was taken out of App.jsx
is it needed for Journal stuff?